stable/nginx-ingress/templates/default-backend-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
```

Comment:
Oh, I guess you could load a different image using the defaultBackend properties. So that might work for some use cases.

It would not give you the ability to load a differently setup pod though. So, like, if I wanted to load a deployment with an nginx container for my branded page, a fluentd sidecar to pull logs, and a prometheus exporter to monitor it.

So I still think the ability to disable its automatic creation is valuable?